### ISSUE TITLE
Detailview + Listview - Sorting by your preferences column (+ on subpanels)

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#874]
Hi,

It will be so nice, if will possible extend the function of subpanels, that in admin i will available to set up for subpanels by which column it will sort the records in subpanel on detailview + if descending or ascending

Same function will also too nice  for listview of records

![contact_subpanels](https://cloud.githubusercontent.com/assets/5773777/12582825/b8d3d418-c43f-11e5-93ef-5e38e6312738.png)

**Comment by ghermans** [comment]
Great suggestion :+1:  !

**Comment by cumanacr** [comment]
Yeah, I like it :+1:

**Comment by Mausino** [comment]
@murdal @gymad @samus-aran i saw that you're currently working on Filter and Saved Search functions #2834 

I am sorry, if i'm spaming you, but i think, that this suggestion will move the SuiteCRM further with your new functions.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Detailview + Listview - Sorting by your preferences column (+ on subpanels)](https://trello.com/c/4apc34Tv/122-detailview-listview-sorting-by-your-preferences-column-on-subpanels)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
